examples/AISKU/README.md

```diff
@@ -0,0 +1,44 @@
+# Application Insights JavaScript SDK
```

Comment:
Can you add a link to these REAM.md files to the one in the examples/ folder, so that they are linked.